presto-main/src/main/java/com/facebook/presto/metadata/FunctionExtractor.java

```diff
@@ -55,6 +58,14 @@ private FunctionExtractor() {}
             return ScalarFromAnnotationsParser.parseFunctionDefinition(clazz);
         }
 
+        if (clazz.isAnnotationPresent(SqlInvokedScalarFunctions.class)) {
```

Comment:
Can we combine the two? I don't see the point to have a different annotation for multiple functions and single function. We use `@ScalarFunction` to annotate both class and method. It's also quite confusing to user since it's very hard to notice the difference of the two (so very easily to use the wrong one).

Reply:
Ok. Removed `SqlInvokedScalarFunctions`.
If a class is not annotated, `FunctionExtractor` will try to find all scalar and sql invoked scalar function that class.